types/phaser.d.ts

```diff
@@ -85833,7 +85833,7 @@ declare namespace Phaser {
         /**
          * Informs Facebook of a custom update that occurred in the game.
          * This will temporarily yield control to Facebook and Facebook will decide what to do based on what the update is.
-         * Once Facebook returns control to the game the plugin will emit an `update` or `upatefail` event.
+         * Once Facebook returns control to the game the plugin will emit an `update` or `updatefail` event.
```

Comment:
I'm assuming this isn't an intentional typo. Your changelog does a good job of calling out all intentional typos (and also indicates that in general you try to replace them).

Reply:
Is there any such thing as an "intentional" typo? :)

Anyway, please do not submit changes to the TypeScript defs file. It's an automatically generated file. Remove this file from your PR please, so I can merge the rest of the changes.

Reply:
Intentional typos are things like:
* using `referer` in APIs because you have to match someone else's mistake
* a changelog saying:
> Fixed documentation misspelling of `referer` to `referrer` 
* A backwards compatibility shim for `referer`
* Spelling/fuzzing tests to show that searches will find / reject a given thing.

In some cases, I'll offer an alternative thing which will suit the needs, (a misspelling of `localhost` can usually be replaced with `localhost_`).

That said, I'll try to drop the things you've identified. I think that for matter I submitted an upstream PR: liabru/matter-js#788